cmd/ui/src/views/Explore/ContextMenu/ContextMenu.tsx

```diff
@@ -72,9 +75,14 @@ const ContextMenu: FC<{ contextMenu: { mouseX: number; mouseY: number } | null;
             <MenuItem onClick={handleSetStartingNode}>Set as starting node</MenuItem>
             <MenuItem onClick={handleSetEndingNode}>Set as ending node</MenuItem>
 
-            <AssetGroupMenuItem assetGroupId={tierZeroAssetGroupId} assetGroupName='High Value' />
-            <AssetGroupMenuItem assetGroupId={ownedAssetGroupId} assetGroupName='Owned' />
-
+            {checkPermission(Permission.GRAPH_DB_WRITE) && [
```

Comment:
Interesting idea to use an array here - I saw you used a fragment in `packages/javascript/bh-shared-ui/src/components/AssetGroupEdit/AssetGroupEdit.tsx` for a similar use case. Any reason we can't do something similar here?